llvm/unittests/ProfileData/InstrProfTest.cpp

```diff
@@ -6,6 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "llvm/ADT/bit.h"
```

Comment:
I don't think this is necessary?

Reply:
yes. removed it.

I tried to use a struct to represent test values rather than using `std::tuple` in the middle, and the header is needed in the middle (if I have need `llvm::endianness`). A struct is a little bit more expressive but chose to use `std::tuple` in order to use `testing::Combine` and `testing::Values` to generate test parameters.